venue/tasks.py

```diff
@@ -5,27 +5,43 @@
 from operator import itemgetter
 
 import rollbar
+import logging
+from logdna import LogDNAHandler
 from celery import chord, shared_task
 from celery.signals import task_failure
+from celery.exceptions import MaxRetriesExceededError
 from constance import config
 from django.conf import settings
 from django.template.loader import get_template
 from django.utils import timezone
 from postmarker.core import PostmarkClient
 from ws4redis.publisher import RedisPublisher
 from ws4redis.redis_store import RedisMessage
+from venue.scrapers.exceptions import ScraperError
 
 from venue.models import (ForumPost, ForumProfile, ForumSite, ForumUserRank, Ranking, Signature, UserProfile)
 from venue.utils import translation_on
 
 
+options = {
+    'app': 'venue',
+    'env': settings.DEPLOYMENT_ENV,
+    'index_meta': True
+}
+handler = LogDNAHandler(settings.LOGDNA_TOKEN, options)
```

Comment:
1. As I see this SDK works with a buffer/flush system, but maybe it's better to use `rsyslog`?
2. I'd like to use settings file instead as you've written and commented.